zrml/global-disputes/src/tests.rs

```diff
@@ -106,10 +111,59 @@ fn add_vote_outcome_works() {
     });
 }
 
+#[test]
+fn add_vote_outcome_fails_with_outcome_mismatch() {
+    ExtBuilder::default().build().execute_with(|| {
+        // create scalar market
+        let market_id = 0u128;
+        let market = market_mock::<Runtime>();
+        Markets::<Runtime>::insert(market_id, &market);
+        GlobalDisputes::push_voting_outcome(
+            &market_id,
+            OutcomeReport::Scalar(0),
+            &ALICE,
+            10 * BASE,
+        )
+        .unwrap();
```

Comment:
Why is this part of the test?

Reply:
Default behaviour to start a global dispute. We need at least one voting outcome to be able to have a token holder vote.